tests/unit_tests/experiment_plans/test_pin_tip_centring.py

```diff
@@ -17,6 +17,11 @@
 )
 
 
+def mock_generator(x, y):
```

Comment:
Could: We are using this function to mock out the `trigger_and_return_pin_tip`, and it gives us a `pin_x` and `pin_y` value. I'd name it something like `get_fake_pin_values` instead